templates/client-podsecuritypolicy.yaml

```diff
@@ -26,7 +26,7 @@ spec:
     {{- if .Values.client.dataDirectoryHostPath }}
     - 'hostPath'
     {{- end }}
-  hostNetwork: false
+  hostNetwork: {{ .Values.client.hostNetwork }}
```

Comment:
qq: does this need to be wrapped by an `if` check to ensure this value is set only is `.Values.client.hostNetwork` is provided in the values file? I suspect it would be set to `null`/`nil` otherwise which I suspect is an invalid value?